tests/integration/nm/ipv4_test.py

```diff
@@ -40,10 +40,19 @@
 RETRY_TIMEOUT = 5
 
 
+def _dict_is_subset(superset, subset):
+    """
+    Code copied from Ignacio Vazquez-Abrams in
+    https://stackoverflow.com/questions/9323749
+    License is CC BY-SA 3.0
+    """
+    return all(item in superset.items() for item in subset.items())
```

Comment:
Not sure if this is needed. Or if that piece of code could be copyrighted. In addition, we need to check if CC license is compatible with LGPL-2.1.

Reply:
I checked out and it is compatible.

Reply:
I checked also. I put it there is to honor the original developer.

Reply:
Sure, I am fine with it.